src/main/java/duke/Init.java

```diff
@@ -0,0 +1,27 @@
+package duke;
+
+public class Init {
+    //greetings
```

Comment:
Avoid using one-word comments. Comments should be descriptive and one-word comments aren't really all that descriptive